assets/src/domain/eventEditor/services/filterState/datetimes/useFilteredDatetimes.ts

```diff
@@ -9,17 +10,39 @@ import sorters from '@sharedEntities/datetimes/predicates/sorters';
 const useFilteredDatetimes = (): Array<Datetime> => {
 	const dates = useDatetimes();
 
-	const { sortBy, datetimesToShow, perPage, pageNumber, setTotal, total, setPageNumber } = useDatesListFilterState();
+	const {
+		datetimesToShow,
+		pageNumber,
+		perPage,
+		searchText,
+		setPageNumber,
+		setTotal,
+		sortBy,
+		total,
+	} = useDatesListFilterState();
 
 	// Flter the list
 	const filteredEntities = useMemo<Array<Datetime>>(() => {
 		return filters({ dates, show: datetimesToShow });
 	}, [dates, datetimesToShow]);
 
+	// search entities
+	const searchResults = useMemo<Array<Datetime>>(() => {
+		if (!searchText) {
+			return filteredEntities;
+		}
```

Comment:
maybe we can move this conditional inside of `entityListSearch`, so that we don't duplicate this logic outside, because the same conditional is applied in `useFilteredTickets.ts `

Reply:
That's already there, it's just to bail early to avoid running the search filter. This will be improved when Filter Bar is extracted.

Reply:
yes, just saw that conditional inside :+1: